flask_server.py

```diff
@@ -63,7 +72,7 @@ def settings():
         for setting in request.form:
             if len(request.form[setting]) > 0:
                 updated_settings[setting] = request.form[setting]
-        saveSettings(updated_settings)
+        save_settings(updated_settings)
 
 
         return render_template("settings.html", folder_path = updated_settings["folder_path"] if "folder_path" in updated_settings else folder_path, cookie_path = updated_settings["cookie_path"] if "cookie_path" in updated_settings else cookie_path)
```

Comment:
Please collapse them both in a single return statement, no reason to have the ternary operator there as well